collections/ansible_collections/purestorage/flashblade/plugins/modules/purefb_network.py

```diff
@@ -135,13 +135,15 @@ def modify_iface(module, blade):
     """Modify Network Interface IP address"""
     changed = True
```

Comment:
Shouldn’t this ‘changed = True’ be removed?

Reply:
No, because we have to have this for check mode. Check mode should always say the module ran successfully

Reply:
No - we need this for check mode. check mode must always return True when running a module